src/Server.hs

```diff
@@ -173,11 +173,12 @@ updateShapes newShapes conn s =
     let polyShapes = toPolymorphics newShapes
         uninitVals = map NS.toTagExpr $ NS.shapes2vals polyShapes $ NS.uninitializedPaths s
         trans' = NS.insertPaths (NS.uninitializedPaths s) uninitVals (NS.transr s)
```

Comment:
At this point, are the only uninitialized paths in the translation the label dimensions? (Or anything else that has to be figured out renderer-side)